dao/db.go

```diff
@@ -41,6 +45,61 @@ func Init() {
 	}
 	rawDB.SetMaxOpenConns(20)
 
+	// Perform database migrations
+	if conf.MigrationsReset {
+		// A new connection needs to be opened because Postgres doesn't allow deleting the database you are connected
+		// too. You can search for "cannot drop the currently open database" error for more information.
+		adminConnection, err := gorm.Open(postgres.Open(dbStringDefaultDb()), &gorm.Config{Logger: l})
+		if err != nil {
+			log.Fatalln(err)
+		}
+
+		// Terminate any other connections to the database, since otherwise Postgres will not allow deleting a database.
+		disconnectSql := fmt.Sprintf(`SELECT pg_terminate_backend(pid) FROM pg_stat_activity WHERE datname = '%s'`, conf.DatabaseName)
+		err = adminConnection.Exec(disconnectSql).Error
+		if err != nil {
+			log.Fatalln(err)
+		}
+
+		// Perform the database deletion.
+		dropDbSql := fmt.Sprintf(`DROP DATABASE %s`, conf.DatabaseName)
+		err = adminConnection.Exec(dropDbSql).Error
+		if err != nil {
+			log.Fatalln(err)
+		}
+
+		// Recreate the database.
+		createDb := fmt.Sprintf(`CREATE DATABASE %s`, conf.DatabaseName)
+		err = adminConnection.Exec(createDb).Error
+		if err != nil {
+			log.Fatalln(err)
+		}
+
+		// Log and exit so that the application can be rerun without the "reset" flag.
+		logging.Log.Info(`The schema "public" has been reset`)
```

Comment:
yes it is true but regard to last changes - I am suggesting change it to "...database XX has been reset"